wordpress-mobile/WordPress-iOS
# Gutenberg DarkMode

PR for Gutenberg DarkMode

![GutenDark](https://user-images.githubusercontent.com/9772967/63637836-8969bd80-c681-11e9-8c09-c96d81f3ff8f.gif)

Known Issues: 
- ToolBar button colors are not changed. Will need some extra work.
- The SafeArea border colors on Landscape can get some more ❤️
- Dynamically changing between Dark and Light mode with the editor open, some parts of the editor won't change accordingly (bottom-sheet and a few elements in the toolbar)

All these will be worked on, on future PRs.

To test:
- `rake dependencies`
- Build and run using Xcode 11 (beta)
- Set the device to Dark Appearance
- Open gutenberg editor
- Check that is dark 🌑 

Test for a solved crash:
- Open gutenberg editor
- Close gutenberg editor
- Go to iOS settings and change the appearance mode.
- Go back to WordPress app
- Check that it does not crash (there was a crash at this point before)
